tests/unit/helpers/cmci_helper.py

```diff
@@ -52,8 +52,10 @@ def stub_records(self, method, resource_type, records, *args, **kwargs):
 
     def stub_cmci(self, method, resource_type, scheme='https', host=HOST, port=PORT,
                   context=CONTEXT, scope=None, parameters='', response_dict=None,
-                  headers={'CONTENT-TYPE': 'application/xml'}, status_code=200, reason='OK',
+                  headers=None, status_code=200, reason='OK',
                   record_count=None, **kwargs):
+        if headers is None:
```

Comment:
Is this not doing the same thing as previously when it was the default in the parameter?